src/scripts/site.js

```diff
@@ -722,161 +818,22 @@ var AppInfo = {};
         });
-        define("slideshow", [componentsPath + "/slideshow/slideshow"], returnFirstDependency);
-        define("objectassign", [componentsPath + "/polyfills/objectassign"], returnFirstDependency);
-        define("focusPreventScroll", [componentsPath + "/polyfills/focusPreventScroll"], returnFirstDependency);
-        define("userdataButtons", [componentsPath + "/userdatabuttons/userdatabuttons"], returnFirstDependency);
-        define("emby-playstatebutton", [componentsPath + "/userdatabuttons/emby-playstatebutton"], returnFirstDependency);
-        define("emby-ratingbutton", [componentsPath + "/userdatabuttons/emby-ratingbutton"], returnFirstDependency);
-        define("listView", [componentsPath + "/listview/listview"], returnFirstDependency);
-        define("indicators", [componentsPath + "/indicators/indicators"], returnFirstDependency);
-        define("viewSettings", [componentsPath + "/viewsettings/viewsettings"], returnFirstDependency);
-        define("filterMenu", [componentsPath + "/filtermenu/filtermenu"], returnFirstDependency);
-        define("sortMenu", [componentsPath + "/sortmenu/sortmenu"], returnFirstDependency);
-        define("idb", [componentsPath + "/idb"], returnFirstDependency);
-        define("sanitizefilename", [componentsPath + "/sanitizefilename"], returnFirstDependency);
-        define("toast", [componentsPath + "/toast/toast"], returnFirstDependency);
-        define("scrollHelper", [componentsPath + "/scrollhelper"], returnFirstDependency);
-        define("touchHelper", [componentsPath + "/touchhelper"], returnFirstDependency);
-        define("appSettings", [componentsPath + "/appSettings"], returnFirstDependency);
-        define("userSettings", [componentsPath + "/usersettings/usersettings"], returnFirstDependency);
-        define("userSettingsBuilder", [componentsPath + "/usersettings/usersettingsbuilder", "layoutManager", "browser"], returnFirstDependency);
-        define("imageUploader", [componentsPath + "/imageuploader/imageuploader"], returnFirstDependency);
-        define("htmlMediaHelper", [componentsPath + "/htmlMediaHelper"], returnFirstDependency);
-        define("viewContainer", [componentsPath + "/viewContainer"], returnFirstDependency);
-        define("dialogHelper", [componentsPath + "/dialogHelper/dialogHelper"], returnFirstDependency);
-        define("serverNotifications", [componentsPath + "/serverNotifications/serverNotifications"], returnFirstDependency);
-        define("skinManager", [componentsPath + "/skinManager"], returnFirstDependency);
-        define("keyboardnavigation", [componentsPath + "/keyboardnavigation"], returnFirstDependency);
-        define("scrollManager", [componentsPath + "/scrollManager"], returnFirstDependency);
-        define("autoFocuser", [componentsPath + "/autoFocuser"], returnFirstDependency);
+        define("userSettingsBuilder", ["components/usersettings/usersettingsbuilder", "layoutManager", "browser"], returnFirstDependency);
```

Comment:
userSettingsBuilder was removed not needed here

Reply:
I need to rebase, it wasn't merged yet when I made the branch.